### PR TITLE
Fix for issue 2635

### PR DESCRIPTION
**Pull request by ShreyaSingh10**
## Description
This fixes the overlapping problem with the WordPress admin sidebar.

## How Has This Been Tested?
Tested with chrome inspector too.

### CODE REVIEW

File: blocks/library/button/editor.scss [hunk 1]
```diff
@@ -15,7 +15,7 @@ $blocks-button__height: 46px;
 		z-index: 2;
 		top: $blocks-button__height + 2px;
 		left: 50%;
-		transform: translateX( -50% );
```
Comment (youknowriad) [inline blocks/library/button/editor.scss]: Which bug is this fixing exactly? and why `13%`?

Comment (ShreyaSingh10) [reply blocks/library/button/editor.scss]: This fix for #2635

**Comment by Soean** [comment]
Issue #2635 is fixed, so we can close this PR.

**Comment by youknowriad** [comment]
Thanks for you work on this PR @ShreyaSingh10 The issue is fixed now.

**Pull request closed without merging**